translations/it/pack/return/rtdwl.json

```diff
@@ -0,0 +1,74 @@
+[
+    {
+        "code": "51001",
+        "name": "Cartucciera",
+        "slot": "Corpo",
+        "text": "Possiedi 2 slot mano aggiuntivi, a cui possono essere associati solo supporti [[Arma]].\nFinché possiedi supporti [[Arma]] associati a 2 o più slot mano, ricevi +1 [willpower]."
```

Comment:
Looks like a comma is missing on this line

Reply:
Hey thank you for the double check!
Should be fixed now :)